sns.tf

```diff
@@ -0,0 +1,24 @@
+resource "aws_sns_topic" "this" {
+  count = var.sns_topic_arn == "" ? 1 : 0
+  name  = var.name
```

Comment:
encryption

Reply:
have a possibility to pass https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sns_topic#kms_master_key_id-1 and if not specified default to SNS aws managed key